python/lancedb/table.py

```diff
@@ -559,7 +559,8 @@ def add(
             The data to insert into the table.
         mode: str
             The mode to use when writing the data. Valid values are
-            "append" and "overwrite".
+            "append", which inserts new rows, and "overwrite", which replaces
+            the entire content of the table with the new rows.
```

Comment:
Drive-by change based on user feedback in discord. In retrospect, `Table.add(mode="overwrite")` is a little confusing, since `add` implies append. Perhaps we should make a separate `Table.overwrite()` method?